libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
```

Comment:
Agreed it doesn't necessarily belong in xml.cpp; I put it there msotly because the existing functions were also there.  Really, though, since it operates on ordinary an QString and doesn't rewquire the Text class otherwise, utils.cpp is where I was thinking the alternative might be.

BTW, my original implementation of this function took a QStringList as an argument and basically looped through them all doing what I am doing here, so we could optionally protect a whole list of tags.  Probably it would be better to just define the list of tags within this function, then still loop in the same way.

Do you want me to make these changes, or are you planning on it?